study.md

```diff
@@ -49,19 +49,23 @@ unique. Write the migration would be used to create a table satisfying these
 requirements.
 
 ```ruby
-# your response here
+generate model pet name:string  breed:string
+
+# then update migration file with
+
+change_column_null :pets, :name, default: true
 ```
 
 ## Explain the Role of Seed Data
 
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+<!-- This is an included method in rails which adds initial data to the database created-->
```

Comment:
This doesn't explain the role of seed data.  Seed data is used to add initial data to a database and is useful when the database is frequently destroyed and recreated during testing.